src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/DefendantResponseStaffNotificationService.java

```diff
@@ -26,18 +27,20 @@
     private final StaffEmailProperties emailProperties;
     private final DefendantResponseStaffNotificationEmailContentProvider emailContentProvider;
     private final DefendantResponseCopyService defendantResponseCopyService;
+    private final DocumentTemplates documentTemplates;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "documentTemplates" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1068)